CheckOut/CheckOut/settings.py

```diff
@@ -25,7 +25,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['checkout-env.eba-9f9crzri.us-east-1.elasticbeanstalk.com', '127.0.0.1', '172.31.40.50']
```

Comment:
may be the incorrect host. 
try "checkout-env.eba-a2fjneg3.us-west-2.elasticbeanstalk.com" if issues arise.